monkey/common/user_roles.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class UserRoles(Enum):
+    """
+    An Enum representing user roles.
+    This Enum represents roles that the user can have. The value
+    of each member is the description of the role
+    """
+
+    ISLAND = "Monkey Island, C&C Server"
```

Comment:
ISLAND is a poor name for a role, as ISLAND is the server users are authenticating too. My suggestion was "IslandInterface", but maybe you can come up with something better